pkgs/development/python-modules/cryptography_vectors/default.nix

```diff
@@ -8,8 +8,8 @@ buildPythonPackage rec {
   pname = "cryptography_vectors";
   version = "2.1.4";
 
-  src = fetchPypi {
-    inherit pname version;
+  src = fetchurl {
+    url = "https://pypi.python.org/packages/de/35/bb4e1e0c182b4f84d9737240d774ec347bea9a7f652e098e4a063c25d24e/${pname}-${version}.tar.gz";
```

Comment:
I don't get a 404 for https://files.pythonhosted.org/packages/source/c/cryptography_vectors/cryptography_vectors-2.1.4.tar.gz.

Reply:
Perhaps that was a temporary problem. Yesterday I had:
```
error: unable to download 'https://files.pythonhosted.org/packages/source/c/cryptography_vectors/cryptography_vectors-2.1.4.tar.gz': HTTP error 404 (curl error: No error)
```
Anyway, I dropped that commit.